x/ibc/04-channel/keeper/keeper_test.go

```diff
@@ -403,22 +404,32 @@ func nextHeader(chain *TestChain) ibctmtypes.Header {
 // TODO: fix tests and replace for real proofs
 
 var (
-	_ commitment.ProofI = validProof{}
+	_ commitment.ProofI = validProof{nil, nil, nil}
 	_ commitment.ProofI = invalidProof{}
 )
 
 type (
-	validProof   struct{}
+	validProof struct {
+		root  commitment.RootI
+		path  commitment.PathI
+		value []byte
+	}
 	invalidProof struct{}
 )
 
 func (validProof) GetCommitmentType() commitment.Type {
 	return commitment.Merkle
 }
 
-func (validProof) VerifyMembership(
+func (proof validProof) VerifyMembership(
```

Comment:
are we still using these proofs?

Reply:
I think we shouldn't be, maybe we can delete them altogether?